dll/win32/browseui/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ list(APPEND SOURCE
     internettoolbar.cpp
     parsecmdline.cpp
     regtreeoptions.cpp
+    settings_cabinet.cpp
     settings.cpp
```

Comment:
```suggestion
    settings.cpp
    settings_cabinet.cpp
```

Reply:
Do you think it would be better to move the cabinet state settings method implementation to `settings.cpp`?

Reply:
yes probably

Reply:
Done 👍